news/2 Fixes/9127.md

```diff
@@ -0,0 +1 @@
+Fix 'ipykernel_launcher' not found when using a global python environment and '.env' file exists.
```

Comment:
will the news tool merge this into the changelog correctly?

Reply:
Yes but you'll have to run it again. Or if you submit your release PR, I can do that before this goes in.